src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/GenerateDocumentHandlerTest.java

```diff
@@ -518,11 +514,11 @@ void should_populate_ftpa_deadline_for_internal_case(YesOrNo yesOrNo) {
         verify(documentGenerator, times(1)).generate(callback);
 
         if (yesOrNo.equals(YesOrNo.YES)) {
-            verify(expectedUpdatedCase).write(FTPA_APPLICATION_DEADLINE_DATE,
-                    EXPECTED_FTPA_DEADLINE_ADA.format(DateTimeFormatter.ofPattern("d MMMM yyyy")));
+            verify(expectedUpdatedCase).write(FTPA_APPLICATION_DEADLINE,
+                    EXPECTED_FTPA_DEADLINE_ADA.format(DateTimeFormatter.ofPattern("d MMM yyyy")));
         } else {
-            verify(expectedUpdatedCase).write(FTPA_APPLICATION_DEADLINE_DATE,
-                    EXPECTED_FTPA_DEADLINE_UK.format(DateTimeFormatter.ofPattern("d MMMM yyyy")));
+            verify(expectedUpdatedCase).write(FTPA_APPLICATION_DEADLINE,
+                    EXPECTED_FTPA_DEADLINE_UK.format(DateTimeFormatter.ofPattern("d MMM yyyy")));
```

Comment:
Same as above

Reply:
Updated LocalDate toString instead of using date formatter